### DISCUSSION TITLE
Naming Virtual Hosts

### DISCUSSION THREAD

**Discussion by dlastnoel** [beyondcode/herd-community#990] [Ideas]
While the function of automatically setting up virtual hosts with .test as the domain is very handy, some development services like Google discourages using .test as domain when using their services. The workaround is to change the vhost name. I think the idea of automatically setting virtual hosts with .test as domain and having the ability to change the domain in the app is great. What are your thoughts? Cheer!

**Comment by mechelon** [comment]
Duplicate of https://github.com/beyondcode/herd-community/discussions/376
